src/index.js

```diff
@@ -1,2 +1,94 @@
 // import _ from 'lodash';
 import './style.css';
+
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+let gameId;
+
+const createGame = async () => {
+  try {
+    const response = await fetch(`${baseURL}/games/`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ name: 'My cool new game' }),
+    });
+    const game = await response.json();
+    // eslint-disable-next-line prefer-destructuring
+    gameId = game.result.split(' ')[3];
+    console.log('Game ID:', gameId);
```

Comment:
Kindly take away every `console.log` statement in your codebase as it is unprofessional to push them to your codebase on Github 😉 It is better they are removed after debugging👍🏼.